test/new_relic/agent/javascript_instrumentor_test.rb

```diff
@@ -158,7 +158,7 @@ def test_browser_timing_header_safe_when_json_dump_fails
 
   def test_config_data_for_js_agent
     nr_freeze_process_time
-    with_config(CAPTURE_ATTRIBUTES => true) do
+    with_config(ATTRIBUTES_ENABLED => true) do
```

Comment:
these tests were all using the old config. Other tests in this file were already using the new config as `ATTRIBUTES_ENABLED`, so these tests were just updated to use that now